.github/workflows/main.yml

```diff
@@ -25,5 +25,17 @@ jobs:
       with:
         ruby-version: ${{ matrix.ruby-version }}
         bundler-cache: true
-    - name: Run the default task
+    - name: Run Tests
       run: bundle exec rake
+
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: 2.6
```

Comment:
Any reason for 2.6?

Reply:
Oh, which leads me to wonder: can we go lower? We may eventually work around the dependency requirements, but perhaps our code should remain compliant with older syntax? :barf: